docs/linters.rst

```diff
@@ -4,40 +4,12 @@ Linters
 .. index:: linters
 
 
-flake8
+ruff
 ------
 
-To run flake8: ::
+To run ruff: ::
```

Comment:
Might be a good time to expand a bit, compared to the old version: explain that we use Ruff to lint and format code, with a sentence or 2 about what Ruff is and link to the official documentation.

Also mention the pre-commit way of running it?

Reply:
Added some documentation, can you check?. Don't know if we need to add a paragraph about pre-commit here since it is explained in the dev documentation and not a linter.